wordpress-mobile/WordPress-iOS
# Comments: add filter tab bar

Ref: #15955

This adds a `FilterTabBar` at the top of the Comments view. 
- `CommentsViewController` is now a `UIViewController` instead of a `UITableViewController`.
- `CommentsViewController` shows the new `CommentsList.storyboard`, which contains a `FilterTabBar` and `UITableView`.
- Selecting different filters doesn't do anything yet.

To test:
- Go to Site > Comments.
- Verify the filter tab bar is not shown.

<kbd><img width="468" alt="no_filter" src="https://user-images.githubusercontent.com/1816888/108910411-0bbe9f00-75e3-11eb-9bbe-3e00ebafacf6.png"></kbd>

- Enable the feature flag.
- Go to Site > Comments.
- Verify the filter tab bar is shown.
- To show tab selection actually works, the tab index is logged out. I'll remove this before merging.
`🔴 tab selected:  1`

<kbd><img width="470" alt="filter" src="https://user-images.githubusercontent.com/1816888/108926499-cb1f4f80-75fb-11eb-9661-fd4c25cd803d.png"></kbd>

PR submission checklist:
